satpy/readers/ahi_hsd.py

```diff
@@ -364,25 +367,44 @@ def __del__(self):
 
     @property
     def start_time(self):
-        """Get the start time."""
-        return datetime(1858, 11, 17) + timedelta(days=float(self.basic_info['observation_start_time']))
+        """Get the scheduled start time."""
+        return self.scheduled_start_time
 
     @property
     def end_time(self):
-        """Get the end time."""
+        """Get the scheduled end time."""
+        return self.scheduled_start_time
+
+    @property
+    def observation_start_time(self):
+        """Get the observation start time."""
+        return datetime(1858, 11, 17) + timedelta(days=float(self.basic_info['observation_start_time']))
+
+    @property
+    def observation_end_time(self):
+        """Get the observation end time."""
         return datetime(1858, 11, 17) + timedelta(days=float(self.basic_info['observation_end_time']))
 
     @property
-    def scheduled_time(self):
+    def scheduled_start_time(self):
         """Time this band was scheduled to be recorded."""
+        return self._modify_obs_time_for_schedule(self.observation_start_time)
+
+    @property
+    def scheduled_end_time(self):
+        """Get the scheduled end time."""
+        return self._modify_obs_time_for_schedule(self.observation_end_time)
+
+    def _modify_obs_time_for_schedule(self, obs_time):
```

Comment:
Can you please explain in the docstring what the scheduled time looks like? It's hard for me to grasp that from the `obs_time.replace()` call

Reply:
Done.